cmake/modules/SwiftSupport.cmake

```diff
@@ -61,6 +61,30 @@ function(get_swift_host_os result_var_name)
   endif()
 endfunction()
 
+# Attempt to get the module triple from the Swift compiler.
```

Comment:
Lets wrap this in a check to see if the variable is set so that we don't re-run the compiler each time we configure:

```cmake
if(NOT Swift_MODULE_TRIPLE)
...
endif()
```

This is what it looks like for the new stdlib build: https://github.com/swiftlang/swift/blob/25252cd6836f512ff820ff995ae8d92ab47076d1/Runtimes/Core/cmake/modules/PlatformInfo.cmake#L7-L20

Reply:
Done.